joomla/joomla-cms
# Make sure automatic title is enabled by default like we had it in 2.5

Pull Request for Issue https://github.com/joomlagerman/joomla/issues/437

### Summary of Changes

Make sure automatic title is enabled by default like we had it in 2.5

### Testing Instructions

[Install this branch](https://github.com/zero-24/joomla-cms/archive/autotitle.zip) and confirm that the setting Automatic Title is enabled for `Popular Articles`, `Recently Added Articles` & `Logged-in Users`

### Expected result

The setting is enabled and shows the generated title instead of the default.

### Actual result

The setting is disabled and just the original title is displayed.

### Documentation Changes Required

none.
